packages/sdk/src/sdk/sdk.ts

```diff
@@ -135,7 +135,7 @@ function initByChain<Symbols extends AssetSymbol, ChainKeys extends ChainKey>(
           return {
             get: async (
               destinationAccount: string,
-              { ethersSigner }: WithdrawGetParams = {},
+              { ethersSigner, originAccount = '' }: WithdrawGetParams = {},
```

Comment:
Is it the same as `const originAccount = signer.getAddress();`?

Reply:
Ha, yes it is, so I don't need to pass the origin account. Good catch, I'll fix it